config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,15 @@
+CarrierWave.configure do |config|
+  if Rails.env.test?
+    config.storage = :file
+    config.enable_processing = false
+  else
+    config.fog_credentials = {
+      provider: 'AWS',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.